CI/samples.ci/client/petstore/java/test-manual/okhttp-gson/api/PetApiTest.java

```diff
@@ -1,564 +0,0 @@
-                            + ": " + progressList);
-                }
-            }
-            prev = progress;
-            index += 1;
-        }
-
-        if (!prev.done) {
-            fail("Last progress item should have done=true: " + progressList);
-        }
-    }
-
-    private static class TestApiCallback<T> implements ApiCallback<T> {
-
-        private final CountDownLatch latch;
-        private final ConcurrentLinkedQueue<Progress> uploadProgress =
-                new ConcurrentLinkedQueue<Progress>();
-        private final ConcurrentLinkedQueue<Progress> downloadProgress =
-                new ConcurrentLinkedQueue<Progress>();
-
-        private boolean done;
-        private boolean success;
-        private ApiException exception;
-        private T result;
-
-        public TestApiCallback(CountDownLatch latch) {
-            this.latch = latch;
-            this.done = false;
-        }
-
-        @Override
-        public void onFailure(ApiException e, int statusCode, Map<String, List<String>> responseHeaders) {
-            exception = e;
-            this.done = true;
-            this.success = false;
-            latch.countDown();
-        }
-
-        @Override
-        public void onSuccess(T result, int statusCode, Map<String, List<String>> responseHeaders) {
-            this.result = result;
-            this.done = true;
-            this.success = true;
-            latch.countDown();
-        }
-
-        @Override
-        public void onUploadProgress(long bytesWritten, long contentLength, boolean done) {
-            uploadProgress.add(new Progress(bytesWritten, contentLength, done));
-        }
-
-        @Override
-        public void onDownloadProgress(long bytesRead, long contentLength, boolean done) {
-            downloadProgress.add(new Progress(bytesRead, contentLength, done));
-        }
-
-        public boolean isDone() {
-            return done;
-        }
-
-        public boolean isSuccess() {
-            return success;
-        }
-
-        public ApiException getException() {
-            return exception;
-        }
-
-        public T getResult() {
-            return result;
-        }
-
-        public List<Progress> getUploadProgress() {
-            return new ArrayList<Progress>(uploadProgress);
-        }
-
-        public List<Progress> getDownloadProgress() {
-            return new ArrayList<Progress>(downloadProgress);
-        }
-    }
-
-    private static class Progress {
-        public final long bytes;
-        public final long contentLength;
-        public final boolean done;
-
-        public Progress(long bytes, long contentLength, boolean done) {
-            this.bytes = bytes;
-            this.contentLength = contentLength;
-            this.done = done;
-        }
-
-        @Override
-        public String toString() {
-            return "<Progress " + bytes + " " + contentLength + " " + done + ">";
-        }
-    }
-}
```

Comment:
Note: this still exists at https://github.com/OpenAPITools/openapi-generator/blob/ensures/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/api/PetApiTest.java